extern/open.php

```diff
@@ -2,10 +2,12 @@
 require_once '../civicrm.config.php';
 require_once 'CRM/Core/Config.php';
 require_once 'CRM/Core/Error.php';
-require_once 'CRM/Utils/Array.php';
+require_once 'CRM/Utils/Type.php';
```

Comment:
@seamuslee001 do we need Type & Rule here now?

Reply:
They are used by the Request function and given that we aren't actually loading the class loader here we can't be certain they are loaded at this point. I don't think Array is used now for this